### DISCUSSION TITLE
Adding a favicon

### DISCUSSION THREAD

**Discussion by turnipsforme** [maximevaillancourt/digital-garden-jekyll-template#182] [Q&A]
Hi, which html file should I add my favicon line of code to? Thanks!

**Comment by maximevaillancourt** [comment]
This one: https://github.com/maximevaillancourt/digital-garden-jekyll-template/blob/40101f01a1b61a68477ef70952c0f529ab33c82c/_includes/head.html

Add your `<link>` tag somewhere between the `<head>` and `</head>` tags.
